tests/v2/validation/configmaps/configmaps.go

```diff
@@ -0,0 +1,36 @@
+package configmaps
+
+import (
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/configmaps"
+	cm "github.com/rancher/rancher/tests/framework/extensions/configmaps"
+)
+
+var(
+	labels = map[string]string{"label1": "autoLabel"}
+	data = map[string]string{"foo": "bar"}
+  )
+
+const(
+	namespace = "default"
+	configMapSteveType = cm.ConfigMapSteveType
```

Comment:
we don't have to add this as a constant since its already defined in the extension github.com/rancher/rancher/tests/framework/extensions/configmaps.